packages/frontend/src/components/UserSettings/UserAttributesPanel/UserAttributeModal.tsx

```diff
@@ -37,7 +37,7 @@ const UserAttributeModal: FC<{
             name: userAttribute?.name || '',
             description: userAttribute?.description,
             users: userAttribute?.users || [],
-            // TODO: add groups
```

Comment:
I think we are missing validation for groups so it's not possible to set them twice.
<img width="1056" alt="Screenshot 2024-01-10 at 10 27 57" src="https://github.com/lightdash/lightdash/assets/9117144/08e7dc69-fd51-4e24-81ef-34a71f143f75">

Users show an error, even though is not in the right place.

<img width="580" alt="Screenshot 2024-01-10 at 10 28 17" src="https://github.com/lightdash/lightdash/assets/9117144/1ec0a0b0-590e-46db-aaa8-455e35d89840">

Reply:
Yea, the validation in this form could use a bit of an update. Even with the users it should not let you pick duplicates instead of giving that error. 

I'll make validation better for groups and look into users too.

Reply:
Done. Updated both form sections to useform validation